dev_guide/builds/buildconfig_environment.adoc

```diff
@@ -0,0 +1,147 @@
+[[dev-guide-build-buildconfig-environment]]
+= BuildConfig Environment
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[NOTE]
+====
+You can also manage environment variables defined in the `BuildConfig` with the
+xref:../../dev_guide/environment_variables.adoc#dev-guide-environment-variables[`oc set env`] command.
+====
+
+[[setting-environment-variables]]
+== Setting Environment Variables
+
```

Comment:
also build_loglevel is a problematic example.  pick something less special, eg "examplevariable=examplevalue"